README.md

```diff
@@ -4,32 +4,27 @@
 
 This suite consists of Loki benchmarks tests for multiple scenarios. Each scenario asserts recorded measurements against a selected profile from the [config](./config) directory:
 
-1. **High Volume Writes**: Measure `QPS`, `p99`, `p50` and `avg` request duration for all 2xx write requests to all Loki distributor and ingester pods.
-2. **High Volume Reads**: Measure `QPS`, `p99`, `p50` and `avg` request duration for all 2xx read requests to all Loki query-frontend, querier and ingester pods.
-3. **High Volume Aggregate**: Measure `QPS`, `p99`, `p50` and `avg` request duration for all 2xx read requests to all Loki query-frontend, querier and ingester pods.
-**...**
-   
+1. **Write benchmarks**:  
+   - High Volume Writes: Measure `CPU`, `MEM` and `QPS`, `p99`, `p50` `avg` request duration for all 2xx write requests to all Loki distributor and ingester pods.
+2. **Read benchmarks:  
```

Comment:
Missing newline before and missing bold-end-syntax `**`.

Reply:
fixed, done